Base/usr/share/man/man1/which.md

```diff
@@ -0,0 +1,25 @@
+## Name
+
+which - shows the full path of commands
+
+## Synopsis
+
+```**sh**
+$ which [options] executable
+```
+
+## Description
+
+`which` prints to stdout the full path of the executable that would have been executed when this argument had been entered at the command prompt.
```

Comment:
This is clearly copied from the Linux man page (https://linux.die.net/man/1/which), including the awkward wording of "prints to stdout the full path of ..." (instead of the correct "prints the full path of ... to stdout"). Please don't do that.